### PR TITLE
DOCS-1982-CE-3.18-k8s

### PR DESCRIPTION
**Pull request by bmckercher123**
Product Version(s):

Issue:

Link to docs preview:

SME review:
- [x] An SME has approved this change. 

DOCS review:
- [x] A member of the docs team has approved this change.

Additional information:

Merge checklist:

- [x] Deploy preview inspected wherever changes were made 
- [x] Build completed successfully
- [x] Test have passed

### CODE REVIEW

**Comment by ctauchen** [comment]
@sujeet-kr Can you confirm support for Kubernetes 1.28? We need this approval ASAP.

File: calico-enterprise/getting-started/compatibility.mdx [hunk 1]
```diff
@@ -94,6 +96,7 @@ Kubernetes version support aligns with [upstream Kubernetes](#kubernetes-kubeadm
 
 | {{prodname}} version | MKE version                          | {{prodname}} support                 | Kubernetes versions |
 | -------------------- | ------------------------------------ | ------------------------------------ | ------------------- |
+| 3.18                 | MKE 3.7.1                            | {{prodname}} CNI with network policy | 1.28                |
```
Comment (sabags) [inline calico-enterprise/getting-started/compatibility.mdx]:
```suggestion
| 3.18                 | MKE 3.7.1                            | {{prodname}} CNI with network policy | 1.27                |
```

File: calico-enterprise/getting-started/compatibility.mdx [hunk 2]
```diff
@@ -62,6 +62,7 @@ Kubernetes version support aligns with [upstream Kubernetes](#kubernetes-kubeadm
 
 | {{prodname}} version | kOps and Kubernetes versions | {{prodname}} support                                         |
 | -------------------- | ---------------------------- | ------------------------------------------------------------ |
+| 3.18                 | 1.25 - 1.28                  | - {{prodname}} CNI with network policy<br />- AWS CNI with {{prodname}} network policy |
```
Comment (sabags) [inline calico-enterprise/getting-started/compatibility.mdx]:
```suggestion
| 3.18                 | 1.26 - 1.28                  | - {{prodname}} CNI with network policy<br />- AWS CNI with {{prodname}} network policy |
```

File: calico-enterprise/getting-started/compatibility.mdx [hunk 3]
```diff
@@ -76,6 +77,7 @@ Kubernetes version support aligns with [upstream Kubernetes](#kubernetes-kubeadm
 
 | {{prodname}} version | Kubernetes/kubeadm versions | {{prodname}} support                 |
 | -------------------- | --------------------------- | ------------------------------------ |
+| 3.18                 | 1.25 - 1.28                 | {{prodname}} CNI with network policy |
```
Comment (sabags) [inline calico-enterprise/getting-started/compatibility.mdx]:
```suggestion
| 3.18                 | 1.26 - 1.28                 | {{prodname}} CNI with network policy |
```

File: calico-enterprise/getting-started/compatibility.mdx [hunk 4]
```diff
@@ -112,6 +115,7 @@ Kubernetes version support aligns with [upstream Kubernetes](#kubernetes-kubeadm
 
 | {{prodname}} version | OpenShift versions for Kubernetes | {{prodname}} support                 |
 | -------------------- | --------------------------------- | ------------------------------------ |
+| 3.18                 | 4.12 - 4.13                       | {{prodname}} CNI with network policy |
```
Comment (sabags) [inline calico-enterprise/getting-started/compatibility.mdx]:
```suggestion
| 3.18                 | 4.12 - 4.14                       | {{prodname}} CNI with network policy |
```

File: calico-enterprise/getting-started/compatibility.mdx [hunk 5]
```diff
@@ -130,6 +134,7 @@ Kubernetes version support aligns with [upstream Kubernetes](#kubernetes-kubeadm
 
 | {{prodname}} version | RKE version | {{prodname}} support                 | Kubernetes versions |
 | -------------------- | ----------- | ------------------------------------ | ------------------- |
+| 3.18                 | 1.4         | {{prodname}} CNI with network policy | 1.28                |
```
Comment (sabags) [inline calico-enterprise/getting-started/compatibility.mdx]:
```suggestion
| 3.18                 | 1.4         | {{prodname}} CNI with network policy | 1.26                |
```

File: calico-enterprise/getting-started/compatibility.mdx [hunk 6]
```diff
@@ -148,6 +153,7 @@ Kubernetes version support aligns with [upstream Kubernetes](#kubernetes-kubeadm
 
 | {{prodname}} version | {{prodname}} support                 | Kubernetes versions |
 | -------------------- | ------------------------------------ | ------------------- |
+| 3.18                 | {{prodname}} CNI with network policy | 1.25 - 1.28         |
```
Comment (sabags) [inline calico-enterprise/getting-started/compatibility.mdx]:
```suggestion
| 3.18                 | {{prodname}} CNI with network policy | 1.26 - 1.28         |
```

**Comment by sabags** [comment]
> @sujeet-kr Can you confirm support for Kubernetes 1.28? We need this approval ASAP.

Yes @ctauchen, kubernetes 1.28 will be supported with 3.18

File: calico-enterprise/getting-started/compatibility.mdx [hunk 7]
```diff
@@ -157,6 +163,7 @@ Kubernetes version support aligns with [upstream Kubernetes](#kubernetes-kubeadm
 
 | {{prodname}} version | TKG version | {{prodname}} support                 | Kubernetes versions    |
 | -------------------- | ----------- | ------------------------------------ | ---------------------- |
+| 3.18                 | 1.6         | {{prodname}} CNI with network policy | 1.23                   |
```
Comment (sujeet-kr) [inline calico-enterprise/getting-started/compatibility.mdx]:
```suggestion
```

File: calico-enterprise/release-notes/index.mdx [hunk 8]
```diff
@@ -50,6 +50,10 @@ For more information, see [Helm installation reference](../reference/installatio
 
 <!-- BELOW ARE 3.18 EP2 features -->
 
+### Support for Kubernetes v1.28
```
Comment (sujeet-kr) [inline calico-enterprise/release-notes/index.mdx:54]: This needs to be updated by Phil.

**Comment by sujeet-kr** [comment]
> @sujeet-kr Can you confirm support for Kubernetes 1.28? We need this approval ASAP.

Yes will be. Thank you for checking.

File: calico-enterprise/getting-started/compatibility.mdx (hunk 2 above)
Comment (bmckercher123) [reply calico-enterprise/getting-started/compatibility.mdx]: Fixed.

File: calico-enterprise/getting-started/compatibility.mdx (hunk 3 above)
Comment (bmckercher123) [reply calico-enterprise/getting-started/compatibility.mdx]: Fixed.

File: calico-enterprise/getting-started/compatibility.mdx (hunk 1 above)
Comment (bmckercher123) [reply calico-enterprise/getting-started/compatibility.mdx]: Fixed.

File: calico-enterprise/getting-started/compatibility.mdx (hunk 4 above)
Comment (bmckercher123) [reply calico-enterprise/getting-started/compatibility.mdx]: Fixed.

File: calico-enterprise/getting-started/compatibility.mdx (hunk 5 above)
Comment (bmckercher123) [reply calico-enterprise/getting-started/compatibility.mdx]: Fixed.

File: calico-enterprise/getting-started/compatibility.mdx (hunk 6 above)
Comment (bmckercher123) [reply calico-enterprise/getting-started/compatibility.mdx]: Fixed.

File: calico-enterprise/getting-started/compatibility.mdx (hunk 7 above)
Comment (bmckercher123) [reply calico-enterprise/getting-started/compatibility.mdx]: Fixed.

File: calico-enterprise/release-notes/index.mdx (hunk 8 above)
Comment (bmckercher123) [reply calico-enterprise/release-notes/index.mdx:54]: I've added this to PR for 3.18-2.

File: calico-enterprise/getting-started/compatibility.mdx (hunk 1 above)
Comment (bmckercher123) [reply calico-enterprise/getting-started/compatibility.mdx]: Fixed

File: calico-enterprise/getting-started/compatibility.mdx (hunk 2 above)
Comment (bmckercher123) [reply calico-enterprise/getting-started/compatibility.mdx]: Fixed.

File: calico-enterprise/getting-started/compatibility.mdx (hunk 3 above)
Comment (bmckercher123) [reply calico-enterprise/getting-started/compatibility.mdx]: Fixed.

File: calico-enterprise/getting-started/compatibility.mdx (hunk 4 above)
Comment (bmckercher123) [reply calico-enterprise/getting-started/compatibility.mdx]: Fixed.

File: calico-enterprise/getting-started/compatibility.mdx (hunk 5 above)
Comment (bmckercher123) [reply calico-enterprise/getting-started/compatibility.mdx]: Fixed.

File: calico-enterprise/getting-started/compatibility.mdx (hunk 6 above)
Comment (bmckercher123) [reply calico-enterprise/getting-started/compatibility.mdx]: Fixed.

File: calico-enterprise/getting-started/compatibility.mdx (hunk 7 above)
Comment (bmckercher123) [reply calico-enterprise/getting-started/compatibility.mdx]: Fixed.

File: calico-enterprise/release-notes/index.mdx (hunk 8 above)
Comment (bmckercher123) [reply calico-enterprise/release-notes/index.mdx:54]: Yes, RN will be handled in the 3.18-2 release PR.

**Review by sabags — APPROVED** [review]

**Pull request merged**